options/linux/include/sys/sysinfo.h

```diff
@@ -0,0 +1,28 @@
+#ifndef _SYS_SYSINFO_H
+#define _SYS_SYSINFO_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct sysinfo {
+	long uptime;
+	unsigned long loads[3];
+	unsigned long totalram;
+	unsigned long freeram;
+	unsigned long sharedram;
+	unsigned long bufferram;
+	unsigned long totalswap;
+	unsigned long freeswap;
+	unsigned short procs;
+	unsigned long totalhigh;
+	unsigned long freehigh;
+	unsigned int mem_unit;
+	char _f[20 - 2 * sizeof(long) - sizeof(int)]; // Padding to 64 bytes according to my man page
```

Comment:
What do the Linux kernel headers say about this?

Reply:
`include/uapi/linux/sysinfo.h` does to what amounts to the same.

The man page specifically says:

> Since Linux 2.3.23 (i386) and Linux 2.3.48 (all architectures) the structure is:
> [...]
> `char _f[20-2*sizeof(long)-sizeof(int)]; /* Padding to 64 bytes */`